ansible/examples/builder.yml

```diff
@@ -919,6 +925,18 @@
           systemd:
             daemon_reload: yes
 
+        - name: Stop jenkins service
+          service:
+            name: jenkins
+            state: stopped
+
+        - name: Kill any errant slave.jar or agent.jar processes
+          shell:
+            cmd: "pkill -f -9 'java.*(slave|agent).jar'"
+          register: result
+          become: true
+          failed_when: result.rc > 1
```

Comment:
Why `> 1` and not i.e. `> 0` ?

Reply:
1 is "no processes matched", which isn't an error; I'm happy if there are no procs to kill.  2/3 are actual errors.